### DISCUSSION TITLE
bf character switching switches dads character instead

### DISCUSSION THREAD

**Discussion by tdog8785** [KadeArchive/Kade-Engine#2154] [Q&A]
Basically I am using the new code for character switching. boyfriend:changeCharacter("pico",600,300) that is the code I am using for it. And when it does switch it changes the dad character instead of the boyfriend one. Same goes for the gf switching.  I am practically stuck on this problem. So any help would be nice.

**Comment by Tikoulla** [comment]
try using bf instead of boyfriend
